BHoM_UI/Components/Engine/SetProperty.cs

```diff
@@ -56,27 +50,19 @@ public class SetPropertyCaller : MethodCaller
         /**** Constructors                ****/
         /*************************************/
 
-        public SetPropertyCaller() : base(typeof(SetPropertyCaller).GetMethod("SetProperty")) { }
+        public SetPropertyCaller() : base(typeof(Engine.Reflection.Modify).GetMethod("PropertyValue")) { }
```

Comment:
I don't see the need to wrap the component around a new method that calls the backend one. 
The name is solved by overriding the `Name` property.